src/webgpu/shader/execution/expression/binary/f32_arithmetic.spec.ts

```diff
@@ -26,89 +26,109 @@ export const d = makeCaseCache('binary/f32_arithmetic', {
       return makeBinaryToF32IntervalCase(lhs, rhs, additionInterval);
     };
 
-    return vectorTestValues(2, false).map(v => {
```

Comment:
I am aware that const and non_const versions are the same. They will be different in the future with my patch to resolve all of the const/non-const issues and leaving it like this makes the merges easier.

Reply:
ack